pkg/app/piped/executor/ecs/rollback.go

```diff
@@ -127,12 +127,46 @@ func rollback(ctx context.Context, in *executor.Input, platformProviderName stri
 	}
 
 	// On rolling back, the scale of desired tasks will be set to 100 (same as the original state).
-	taskSet, err := client.CreateTaskSet(ctx, *service, *td, targetGroup, 100)
+	taskSet, err := client.CreateTaskSet(ctx, *service, *td, primaryTargetGroup, 100)
 	if err != nil {
 		in.LogPersister.Errorf("Failed to create ECS task set %s: %v", *serviceDefinition.ServiceName, err)
 		return false
 	}
 
+	// Reset routing
+	routingTrafficCfg := provider.RoutingTrafficConfig{
+		{
+			TargetGroupArn: *primaryTargetGroup.TargetGroupArn,
+			Weight:         100,
+		},
+		{
+			TargetGroupArn: *canaryTargetGroup.TargetGroupArn,
+			Weight:         0,
+		},
+	}
```

Comment:
```suggestion
	routingTrafficCfg := provider.RoutingTrafficConfig{
		{
			TargetGroupArn: *primaryTargetGroup.TargetGroupArn,
			Weight:         100,
		},
	}
```
Can we do like this? 🤔

Reply:
- I tried to do like this, but `canaryTargetGroup` not change 🤔 🤔 🤔

![image](https://github.com/pipe-cd/pipecd/assets/40441000/7f63130a-d7a6-452c-8407-a81bf26d74e3)

Reply:
- @anhpnv Thanks 🙏